hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManagerImpl.java

```diff
@@ -452,7 +452,7 @@ public boolean addAcl(OzoneObj obj, OzoneAcl acl) throws IOException {
         if (LOG.isDebugEnabled()) {
           LOG.debug("Add acl failed.", ex);
         }
-        return false;
+        throw new OMException(volume, ResultCodes.ACL_ADD_FAILED);
```

Comment:
Why this is only done for volume not for bucket and keys?
only in case of Volume there would be this Result Code, but won't be there for bucket and keys.
A general observation for bucket and keys, if the ACL entry exists already it returns false, but I think in case of Volume there is nothing of that sort. In fact it looks like post this change it can either return a true or an exception

Reply:
Thank you for your suggestion. I did not notice the bucket and keys and will try to modify it.